src/views/Peers.tsx

```diff
@@ -99,11 +109,28 @@ export const Peers = () => {
             message.loading({ content: 'Deleting...', key: deleteKey, style });
         } else if (deletedPeer.success) {
             message.success({ content: 'Peer has been successfully removed.', key: deleteKey, duration: 2, style });
+            dispatch(peerActions.resetDeletedPeer(null))
         } else if (deletedPeer.error) {
             message.error({ content: 'Failed to delete peer. You might not have enough permissions.', key: deleteKey, duration: 2, style  });
+            dispatch(peerActions.resetDeletedPeer(null))
         }
     }, [deletedPeer])
 
+    const saveGroupsKey = 'saving_groups';
+    useEffect(() => {
+        const style = { marginTop: 85 }
+        if (savedGroups.loading) {
+            message.loading({ content: 'Updating groups...', key: saveGroupsKey, style });
+        } else if (savedGroups.success) {
+            message.success({ content: 'Groups updated with success!', key: saveGroupsKey, duration: 2, style });
+            setUpdateGroupsVisible({} as Peer, false)
+            dispatch(peerActions.resetSavedGroups(null))
+        } else if (savedGroups.error) {
+            message.error({ content: 'Error! Something wrong to update groups.', key: saveGroupsKey, duration: 2, style  });
+            dispatch(peerActions.resetSavedGroups(null))
+        }
+    }, [savedGroups])
```

Comment:
Codacy has a fix for the issue: [Replace `····},·[savedGroups])` with `··},·[savedGroups]);`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
  }, [savedGroups]);
```